README.md

```diff
@@ -1,12 +1,23 @@
 # Concurrent Programming with Effect Handlers 
+<!-- [![Build Status](https://travis-ci.org/ocamllabs/ocaml-effects-tutorial.svg?branch=master)](https://travis-ci.org/ocamllabs/ocaml-effects-tutorial)    -->
+[![Build Status](https://github.com/tanelso2/ocaml-effects-tutorial/actions/workflows/prs.yml/badge.svg)](https://github.com/tanelso2/ocaml-effects-tutorial/actions/workflows/prs.yml)
```

Comment:
This still refers to your fork, could you update this to `ocamllabs` please?

Reply:
Done